### ISSUE TITLE
Commas are stripped off in latex

### ISSUE THREAD

**Issue by pauleil** [ndl/wiki_external_filter#5]
How do I make commas (e.g., "x,y") show up?
I tried {{latex(${x,y})}} and {{latex(${x{,}y})}}
but in both cases I get simply "xy".
I am using Redmine 1.1.2 w/Textile (i.e., default setup).

**Comment by ndl** [comment]
Please follow the steps described in the installation documentation at https://www.ndl.kiev.ua/content/redmine-wiki-external-filter-plugin - pay special attention to the 6th step (Redmine core patching for arguments preservation).

I'm closing the issue for now, please reopen it if patching does not help.

**Comment by pauleil** [comment]
I followed your instructions (i.e., applied the patch for 1.0.2 and skipped to step 7, ignoring the rest of step 6).
It doesn't help. I am using Textile.
Anything else I should try?

**Comment by pauleil** [comment]
I don't have the permissions to reopen this issue.

**Comment by ndl** [comment]
Sorry for delay with the answer ... Try replacing the command you use for processing the input with latex by something like `cat > /tmp/test.txt` and examine that file after macro is rendered. This way you should be able to see what exactly is passed as the input to the latex package.

**Comment by pauleil** [comment]
I'm not sure if I understand your instructions. Your plugin uses latex installed on my system to generate the output, right? What command are you referring to? Could you be more specific?
Or are you referring to somehow passing an argument to {{latex(${x,y})}}, to save the output in test.txt?

**Comment by ndl** [comment]
> Your plugin uses latex installed on my system to generate the output, right?

Not exactly: my plugin with its default config does not support latex at all.

From your question I assumed that you've modified its config to add support for the latex (though I've not tried it, it should be more or less trivial), and the command I refer to in my previous comment was the command that should have been added to wiki_external_filter config to add latex support.

If you have not modified default wiki_external_filter config in any way to add latex support - then you should not expect it to work at all ...
